sql/moz-fx-data-shared-prod/firefox_desktop_derived/review_checker_clients_v1/query.sql

```diff
@@ -3,6 +3,9 @@ WITH shopping_metrics AS (
     metrics.uuid.legacy_telemetry_client_id AS legacy_client_id,
     client_info.client_id AS client_id,
     DATE(submission_timestamp) AS submission_date,
+    mozfun.stats.mode_last(
+      ARRAY_AGG(metrics.uuid.legacy_telemetry_profile_group_id ORDER BY submission_timestamp)
```

Comment:
I believe the `ORDER BY` might be a noop here since `stats.mode_last` sorts the ids by frequency: https://github.com/mozilla/bigquery-etl/blob/464f9e183eec8fb124db3c44d4f716b34a24bb64/sql/mozfun/stats/mode_last/udf.sql#L7

Reply:
Correct, it's really just there in case of ties